frontend/src/pages/CreateVotingPage.jsx

```diff
@@ -16,12 +17,13 @@ export function CreateVotingPage() {
     mode: "onBlur",
   });
 
-  const onSubmit = function (data) {
+  const onSubmit = async function (data) {
     const splitTime = data.time.split(":").map(Number);
-    const votingTime = splitTime[0] * 86400 + splitTime[1] * 3600 + splitTime[2] * 60 + splitTime[3];
-    const candidates = data.candidates.filter(candidate => candidate);
-    // TODO: api call
-    console.log(data);
+    const votingTimeInSeconds = splitTime[0] * 86400 + splitTime[1] * 3600 + splitTime[2] * 60 + splitTime[3];
+    const candidateNames = data.candidates.filter(candidate => !!candidate);
+    const formatedData = { name: data.name, description: data.description, votingTimeInSeconds, candidateNames };
+    const result = await createElection(formatedData);
```

Comment:
add error handling here as well. Also if we had success, I think we should navigate to active-votings page. also disable the create button once the API was called with some sort of isLoading state